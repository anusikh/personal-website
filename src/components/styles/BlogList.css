.BlogList__Main {
  margin: 0 1.4rem 1.4rem 1.4rem;
}

@media only screen and (max-width: 800px) {
  .BlogCard__Date {
    display: none;
  }
}
