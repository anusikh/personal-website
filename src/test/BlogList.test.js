import React from "react";
import renderer from "react-test-renderer";
import BlogList from "../components/BlogsList";

it("Henlo", () => {
  const component = renderer.create(<BlogList />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
