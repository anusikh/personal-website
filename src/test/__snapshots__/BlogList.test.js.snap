// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Henlo 1`] = `
<div
  className="BlogList__Main"
/>
`;
